README.md

```diff
@@ -0,0 +1,8 @@
+# nova-tinymce-field
+This package is a Nova WYSIWIG Field that uses TinyMCE, which has an extensive set of configuration/plugins.
+
+### Installation
+- `composer require bythepixel/nova-tinymce-field`
```

Comment:
Add the "publish" command also?

Reply:
@rreynier Yup, added it. Looks like this package doesn't have a "tag" so we have to publish like this, correct?
```
php artisan vendor:publish --provider="Bythepixel\\NovaTinymceField\\FieldServiceProvider"
```

Reply:
Correct.